Automattic/wp-calypso
# Site Profiler: add tooltip to the hosting provider if its popular CDN

Related to #82241

## Proposed Changes

* Adding a tooltip if it's the hosting provider is a popular CDN we recognized. If we have support_url from the API, we'll provide links in the popup, otherwise, it'll be text only.
<img width="1035" alt="Screen Shot 2023-09-28 at 3 24 09 PM" src="https://github.com/Automattic/wp-calypso/assets/4074459/3c884c11-8d62-4ad1-9ef0-eae6308c3af7">



## Testing Instructions



* Go to http://calypso.localhost:3000/site-profiler
* Type in a domain that's using one of the popular CDNs, you can find one of them from the diff here: D123383-code
* Check if there's an information icon showing next to the hosting provider.
* Click it and see if the tooltips shows up, click again to hide it
* Try to see if there's a `contact [hosting_provider_name]` link in the tooltip like the screenshot above
* Click on the link and see if it brings you to the support page of the hosting provider
* Check other domains and make sure there's no regression for other conditions.
* For now, the hosting provider name will only shown as a link if `home_url` is provided, which is not implemented from the backend yet.

## Pre-merge Checklist
